deck.json

```diff
@@ -1358,6 +1358,40 @@
                 "health"
             ]
         },
+        {
+            "__type__": "Note",
+            "data": "",
+            "fields": [
+                "How did South Korea flatten the curve?(tentative March 2020)",
+                "<div>- Quick emergency response</div><div>- Testing broadly and quickly</div><div>- Tracing</div><div>- Public trust and cooperation</div>",
+                "Fisher, Max, and Choe Sang-hun. “How South Korea Flattened the Curve.”&nbsp;<i>The New York Times</i>, The New York Times, 23 Mar. 2020, www.nytimes.com/2020/03/23/world/asia/coronavirus-south-korea-flatten-curve.html.&nbsp;",
+                "\n<div>South Korea's Covid-19 Pandemic Response</div>",
+                ""
+            ],
+            "flags": 0,
+            "guid": "d@6:wMopFe",
+            "newlyAdded": false,
+            "note_model_uuid": "f0dd73fe-6576-11ea-923b-ac72899d63fe",
+            "tags": [
+                "health"
+            ]
+        },
+        {
+            "__type__": "Note",
+            "data": "",
+            "fields": [
+                "How did South Korea flatten the curve?(tentative March 2020)<div><br></div><div><div>- {{c1::Quick emergency response}}</div><div>- {{c2::Testing broadly and quickly}}</div><div>- {{c3::Tracing}}</div><div>- {{c4::Public trust and cooperation}}</div></div>",
+                "",
+                "Fisher, Max, and Choe Sang-hun. “How South Korea Flattened the Curve.”&nbsp;<i>The New York Times</i>, The New York Times, 23 Mar. 2020, www.nytimes.com/2020/03/23/world/asia/coronavirus-south-korea-flatten-curve.html.&nbsp;"
+            ],
+            "flags": 0,
+            "guid": "y$;(|]j?u*",
+            "newlyAdded": false,
+            "note_model_uuid": "ac8b4554-6d1d-11ea-86e7-ac72899d63fe",
+            "tags": [
+                "health"
+            ]
+        },
```

Comment:
As I suggested at #13, the alternative, shorter questions were meant to replace these, that have a longer response, in order to follow [the minimum information principle](https://www.supermemo.com/en/articles/20rules). So these questions can safely be deleted, as the information they contain is covered in the ones below.

Reply:
I viewed the shorter cards as complimentary rather than as replacement. Ideally, you would learn the individual cards before [chunking them](https://en.wikipedia.org/wiki/Chunking_(psychology)) together into a cohesive whole. 

Obviously, there are limit. Too much information makes cards difficult to recall.

It also hopefully helps with reducing the effects of individual cards memorized instead of concepts.

Reply:
I very much prefer to avoid, to the extent possible, long answers or lists in this deck, very much in the tune of the example at [point 4 here](url). But of course, your personal version of the deck can contain this question no problem.

Reply:
Link doesn't work, but I understand your disagreement on the merit of long answer. I dislike having difference between personal and official, but it is what it is.

I'll remove the note in question.

Reply:
It's not ideal, and I appreciate that you see the reason behind it.
In the meantime I've added questions about social distancing and contact tracing, so that they're defined by the time the questions about South Korea arrive.

Also, apart from South Korea it's been Taiwan, Singapore and Hong Kong as well that have done a great job in containing and averting the kinds of tragedies we see elsewhere. I imagine some things they did were similar to South Korea, and some different. I just finished reading [this transcript](https://www.democracynow.org/2020/4/3/taiwan_coronavirus_response), for example, in case you want to check it out. Maybe for another pull request, if you wish; it might make sense to merge the South Korea questions with the ones about Taiwan and the other two countries, and some of the information warrants new questions too.